package.json

```diff
@@ -86,11 +86,11 @@
     "build:client": "react-scripts build",
     "build:server": "NODE_ENV=production webpack --progress --mode production --config config/webpack.server.config.js",
     "codegen": "graphql-codegen -r dotenv/config dotenv_config_path=.env.development.local  --config codegen.yml",
-    "test": "TZ=Europe/Helsinki react-scripts test --env=jest-environment-jsdom-sixteen",
+    "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
     "ci": "CI=true yarn test --verbose --coverage",
-    "test:staged": "yarn test --watchAll=false --findRelatedTests",
+    "test:staged": "yarn test --watchAll=false --findRelatedTests --runInBand --forceExit",
```

Comment:
why is --forceExit needed?

Reply:
it was a fix to the TZ issue, but i cherry picked the Jaakkos branch, and this line was merged incorrectly.. i ll fix